lib/cog/chat/slack/template_processor.ex

```diff
@@ -90,8 +90,8 @@ defmodule Cog.Chat.Slack.TemplateProcessor do
   end
   defp process_directive(%{"name" => "list_item", "children" => children}, bullet: bullet),
     do: "   #{bullet} #{Enum.map_join(children, &process_directive/1)}\n"
-  defp process_directive(%{"name" => "link", "url" => url}, _) do
-    "#{url}"
+  defp process_directive(%{"name" => "link", "text" => text, "url" => url}, _) do
+    "<#{url}|#{text}>"
```

Comment:
I could also be more explicit, that might make things clearer, but the behavior would remain the same.